integration/pom.xml

```diff
@@ -29,7 +29,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>3.8.1</version>
+      <version>${junit.version}</version>
```

Comment:
Hm, I just noticed that in addition to here in dependencies, junit is also defined as a dependency of maven-antrun-plugin below. Should that be updated as well perhaps?